Lib/multiprocessing/util.py

```diff
@@ -102,6 +102,27 @@ def log_to_stderr(level=None):
     _log_to_stderr = True
     return _logger
 
+
+# Abstract socket support
+
+def _platform_supports_abstract_sockets():
+    if "linux" in sys.platform:
```

Comment:
sys.platform should be exactly the string "linux" on Linux. Why checking for substring?

Reply:
I have seen 'linux2' if I recall correctly.

Reply:
Are you living in the 90s? :-D linux2 is gone since Python 3.3: https://docs.python.org/dev/library/sys.html#sys.platform (I changed that when Linux kernel 3.0 was released.)

Reply:
I am a nostalgic, what can I say :)